### PR TITLE
[GCI-179]: Add relevant tags to OpenMRS Add-ons modules

### PR DESCRIPTION
**Pull request by f4ww4z**
https://issues.openmrs.org/browse/GCI-179

I added tags for add-ons with no tags previously and added another tag to other add-ons.

### CODE REVIEW

File: src/main/resources/add-ons-to-index.json [hunk 1]
```diff
@@ -894,6 +921,9 @@
       "uid": "org.openmrs.module.exportccd",
       "type": "OMOD",
       "name": "Export CCD Module",
+      "tags": [
+        "export", "ccd"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: It's not helpful to add a tag that only is used for one addon, and matches a word in the module's name. So remove "ccd". [Edit: later on I see there's another CCD, so it's okay to leave this]
Personally I don't think that "export" or "import" is very helpful if they are just repeating something that's in the title, so I would remove that (and I'll repeat this comment).

File: src/main/resources/add-ons-to-index.json [hunk 2]
```diff
@@ -1032,7 +1062,7 @@
       "type": "OMOD",
       "name": "Form Data Export",
       "tags": [
-        "form-entry"
+        "form-entry", "export"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "export"

File: src/main/resources/add-ons-to-index.json [hunk 3]
```diff
@@ -1114,7 +1144,7 @@
       "type": "OMOD",
       "name": "Form Import Export",
       "tags": [
-        "form-entry"
+        "form-entry", "import", "export"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "export" and "import"

File: src/main/resources/add-ons-to-index.json [hunk 4]
```diff
@@ -1223,6 +1256,9 @@
       "uid": "org.openmrs.module.hl7query",
       "type": "OMOD",
       "name": "HL7 Query",
+      "tags": [
+        "export", "hl7"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "export"

File: src/main/resources/add-ons-to-index.json [hunk 5]
```diff
@@ -1391,6 +1427,9 @@
       "uid": "org.openmrs.module.htmlwidgets",
       "type": "OMOD",
       "name": "HTML Widgets",
+      "tags": [
+        "widgets"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "widgets", it doesn't give any useful info to a user

File: src/main/resources/add-ons-to-index.json [hunk 6]
```diff
@@ -1419,6 +1458,9 @@
       "uid": "org.openmrs.module.i2b2export",
       "type": "OMOD",
       "name": "i2b2 Export",
+      "tags": [
+        "export"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "export"
Add "interoperability"

File: src/main/resources/add-ons-to-index.json [hunk 7]
```diff
@@ -1722,6 +1779,9 @@
       "uid": "org.openmrs.module.mdrtbpatientchartwidgets",
       "type": "OMOD",
       "name": "Mdrtb Patient Chart Widgets",
+      "tags": [
+        "widgets"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "widgets"

File: src/main/resources/add-ons-to-index.json [hunk 8]
```diff
@@ -1988,6 +2051,9 @@
       "uid": "org.openmrs.module.ncd",
       "type": "OMOD",
       "name": "NCD",
+      "tags": [
+        "hl7"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "hl7"

File: src/main/resources/add-ons-to-index.json [hunk 9]
```diff
@@ -2040,6 +2106,9 @@
       "status": "INACTIVE",
       "type": "OMOD",
       "name": "Obs Group Export",
+      "tags": [
+        "export"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "export"

File: src/main/resources/add-ons-to-index.json [hunk 10]
```diff
@@ -2171,6 +2243,9 @@
       "status": "INACTIVE",
       "type": "OMOD",
       "name": "Patient Chart Widgets",
+      "tags": [
+        "widgets"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "widgets"

File: src/main/resources/add-ons-to-index.json [hunk 11]
```diff
@@ -2264,6 +2342,9 @@
       "status": "INACTIVE",
       "type": "OMOD",
       "name": "Patient Summary",
+      "tags": [
+        "notes"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "notes"

File: src/main/resources/add-ons-to-index.json [hunk 12]
```diff
@@ -2731,6 +2812,9 @@
       "status": "INACTIVE",
       "type": "OMOD",
       "name": "Restrict By Role",
+      "tags": [
+        "role"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "role" but add "access-control"

File: src/main/resources/add-ons-to-index.json [hunk 13]
```diff
@@ -2766,6 +2853,9 @@
       "uid": "org.openmrs.module.rolebasedhomepage",
       "type": "OMOD",
       "name": "Role-Based Homepage",
+      "tags": [
+        "role"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: change this to "access-control"

File: src/main/resources/add-ons-to-index.json [hunk 14]
```diff
@@ -2824,6 +2917,9 @@
       "uid": "org.openmrs.module.sdmxhdintegration",
       "type": "OMOD",
       "name": "SDMX-HD Integration",
+      "tags": [
+        "export"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Change to "interoperability"

File: src/main/resources/add-ons-to-index.json [hunk 15]
```diff
@@ -2944,6 +3046,9 @@
       "uid": "org.openmrs.module.spreadsheetimport",
       "type": "OMOD",
       "name": "Spreadsheet Import",
+      "tags": [
+        "import"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "import"

File: src/main/resources/add-ons-to-index.json [hunk 16]
```diff
@@ -3605,6 +3710,9 @@
       "uid": "org.openmrs.module.reference-application-module",
       "type": "OMOD",
       "name": "Reference Application Module",
+      "tags": [
+        "reference-application"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Also add this same tag to:
* Reference Metadata
* Reference Demo Data

File: src/main/resources/add-ons-to-index.json [hunk 17]
```diff
@@ -3904,6 +4015,9 @@
       "uid": "org.openmrs.module.visit-notes-analysis-module",
       "type": "OMOD",
       "name": "Visit Notes Analysis Module",
+      "tags": [
+        "notes"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "notes"

File: src/main/resources/add-ons-to-index.json [hunk 18]
```diff
@@ -3921,7 +4035,7 @@
       "type": "OMOD",
       "name": "Data Import Tool",
       "tags": [
-        "data-management"
+        "data-management", "import"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "import"

File: src/main/resources/add-ons-to-index.json [hunk 19]
```diff
@@ -4024,6 +4138,9 @@
       "uid": "org.openmrs.module.omod-export-module",
       "type": "OMOD",
       "name": "Omod Export Module",
+      "tags": [
+        "export"
```
Comment (djazayeri) [inline src/main/resources/add-ons-to-index.json]: Get rid of "export". Add "sysadmin" (or something else similar if there's already a label like that)

File: src/main/resources/add-ons-to-index.json (hunk 16 above)
Comment (f4ww4z) [reply src/main/resources/add-ons-to-index.json]: It was already done by previous commits @djazayeri

**Comment by f4ww4z** [comment]
I have completed your requested changes @djazayeri , please take a look. Thanks

**Comment by djazayeri** [comment]
Great thanks! I will merge this now.

**Pull request merged**